### PR TITLE
refactor: events card and leaderboard design and font

### PR DESCRIPTION
**Pull request by jevil25**

### CODE REVIEW

File: src/pages/events/index.tsx [hunk 1]
```diff
@@ -306,7 +306,7 @@ const Events: NextPage<{ data: PublishedEventsQuery['publishedEvents'] }> = ({
               ) : (
                 <div
                   data-scroll
-                  className="text-center text-white text-2xl font-bold py-10"
+                className={`${styles.glitch} text-center text-white text-2xl py-10`}
                 >
                   No events found
                 </div>
```
Comment (nagarajpandith) [inline src/pages/events/index.tsx:312]: fallback is placed at bottom left, must be centered vertically and horizontally
Screenshot 2024-02-01 at 12 59 56 PM

File: src/pages/events/index.tsx [hunk 2]
```diff
@@ -114,7 +114,7 @@ const Events: NextPage<{ data: PublishedEventsQuery['publishedEvents'] }> = ({
 
   return (
     <div
-      className="overflow-x-hidden vikingHell"
+      className="overflow-x-hidden black-ops"
       style={{ willChange: 'transform' }}
     >
       <LocomotiveScrollProvider
```
Comment (nagarajpandith) [inline src/pages/events/index.tsx:120]: locomotive has been removed from landing and other pages due to scroll hijacking and loading issues.
either remove smooth scroll from here or please setup [studio-freight/lenis](https://github.com/studio-freight/lenis).

**Review by nagarajpandith — CHANGES_REQUESTED** [review]
resolve the 2 comments

File: src/pages/events/index.tsx (hunk 2 above)
Comment (jevil25) [reply src/pages/events/index.tsx]: i changed it with lenis scroll

**Comment by jevil25** [comment]
I resolved the 2 changes

**Review by nagarajpandith — APPROVED** [review]

**Pull request merged**